ui/page/dexclient/add_dex_modal.go

```diff
@@ -74,6 +74,8 @@ func (md *addDexModal) OnResume() {
 }
 
 func (md *addDexModal) Handle() {
+	md.modal.SetDisabled(md.isSending)
```

Comment:
Where is the value of `md.isSending` modified? That's where `md.modal.SetDisabled` should be called too.